src/zjs_sensor.c

```diff
@@ -174,7 +174,7 @@ static void zjs_sensor_set_state(jerry_value_t obj, enum sensor_state state)
         return;
     }
 
-    // update state property and trigger onstatechange event
+    // update state property and trigger onactivate event if neccessary
```

Comment:
it's already triggering it in the old code, that's why it didn't show up in the diff, at the end of the func it does that